src/TransactionController.ts

```diff
@@ -6,7 +6,14 @@ import { BNToHex, fractionBN, hexToBN, normalizeTransaction, safelyExecute, vali
 const EthQuery = require('eth-query');
 const Transaction = require('ethereumjs-tx');
 const random = require('uuid/v1');
-const { addHexPrefix, bufferToHex } = require('ethereumjs-util');
+const { addHexPrefix, bufferToHex, BN } = require('ethereumjs-util');
+
+const NETWORK_API_URLS: any = {
```

Comment:
Do we need this? I think we can get the string name of the current network from the NetworkController, so we could derive these instead of maintaining a list. Both are equally as brittle.

Reply:
Also, for reference, this type should not be `any`, it should be `{ [networkId: number]: string }`.